testing/tests/i18n.rs

```diff
@@ -0,0 +1,58 @@
+#![cfg(feature = "i18n")]
+
+use askama::i18n::{langid, Locale};
+use askama::Template;
+
+askama::i18n::load!(LOCALES);
```

Comment:
I think this should look like `static LOCALES: _ = askama::i18n::load!();` -- what is preventing us from doing something more like that?

Reply:
this is the expansion of line 6:

```rust
static LOCALES: ::askama::i18n::fluent_templates::once_cell::sync::Lazy<
    ::askama::i18n::fluent_templates::StaticLoader,
> = ::askama::i18n::fluent_templates::once_cell::sync::Lazy::new(|| {
    mod fluent_templates {
        pub use ::askama::i18n::fluent_templates::*;
        pub mod once_cell {
            pub mod sync {
                pub use ::askama::i18n::Unlazy as Lazy;
            }
        }
    }
    pub static LOCALES: fluent_templates::once_cell::sync::Lazy<
        fluent_templates::StaticLoader,
    > = fluent_templates::once_cell::sync::Lazy::new(|| {
        static CORE_RESOURCE: fluent_templates::once_cell::sync::Lazy<
            Option<fluent_templates::fluent_bundle::FluentResource>,
        > = fluent_templates::once_cell::sync::Lazy::new(|| { None });
....
```
...and so on

...so the expansion handles the variable setting. Because the type can't be elided ([it being static](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=45450a94c14613c85fa16a2a13db81da)),  that would be requiring the end user to write like so:

```rust
static LOCALES: askama::18n::fluent_templates::once_cell::Lazy<askama::i18n::fluent_templates::StaticLoader> = askama::i18n::load!();

// with sane re-exports
static LOCALES: askama::i18n::LazyStaticLoader = askama::i18n::load!();
```



OP is being consistent, in a sense, with the pattern used by the wrapped `fluent_templates::static_loader` macro, which is documented like so:


```rust
//! ### Basic Example
//! ```
//! fluent_templates::static_loader! {
//!     // Declare our `StaticLoader` named `LOCALES`.
//!     static LOCALES = {
//!         // The directory of localisations and fluent resources.
//!         locales: "./tests/locales",
//!         // The language to falback on if something is not present.
//!         fallback_language: "en-US",
//!         // Optional: A fluent resource that is shared with every locale.
//!         core_locales: "./tests/locales/core.ftl",
//!     };
//! }
//! # fn main() {}
//! ```
```

...in our case, though, the wrapper handles the locales and other settings by means of reading the i18n.toml file, if i'm reading things correctly

I think it would be worth some follow-up after this PR is merged to see if it would be nice to "de-abstract" a touch the underlying fluent engine. The scenario I have in mind is that an "actix-web + askama" project should be internationalised by turning it into an "actix-web + askama + fluent" and not "actix-web + fluent + askama-with-i18n". That probably doesn't make sense, but this conversation is a bit out of scope, so I'll clarify when it comes time to put on the polish.

Reply:
I think `static LOCALES: askama::i18n::LazyStaticLoader = askama::i18n::load!();` looks better, I prefer it over the more opaque/comprehensive wrapper macro.